### PR TITLE
Cassandra 16296 ccm bug

### PR DESCRIPTION
**Pull request by bereng**

### CODE REVIEW

**Comment by bereng** [comment]
CI [j8](https://app.circleci.com/pipelines/github/bereng/cassandra/207/workflows/e6f08457-290d-4539-83c1-bba82618a0cb): LGTM with some flakys
CI [j11](https://app.circleci.com/pipelines/github/bereng/cassandra/207/workflows/fc1f2a9c-29a4-42a9-b581-64252430eef0): Weird unit failure that passes locally but should be unrelated, consistency test passes locally.

**Comment by bereng** [comment]
After the latest change in CCM:
- [j11](https://app.circleci.com/pipelines/github/bereng/cassandra/208/workflows/1c918162-242d-4a26-b4b1-882c4893e6d2): The unit tests are flaky, pass locally and can't be related. The dtest passes locally.
- [j8](https://app.circleci.com/pipelines/github/bereng/cassandra/208/workflows/ac30f2dd-369e-4ccc-82ba-1801b5149bed): Same as above.

This latest run had more failures but given the unit tests were flaky as well I think it lgtm and I am going to attribute it to the env.

**Review by ekaterinadimitrova2 — APPROVED** [review]

**Comment by ekaterinadimitrova2** [comment]
I think the failing tests are already known flaky and not related to the patch which seems straightforward to me.

**Review by ekaterinadimitrova2 — APPROVED** [review]

File: replace_address_test.py [hunk 1]
```diff
@@ -78,6 +78,8 @@ def _do_replace(self, same_address=False, jvm_option='replace_address',
                     extra_jvm_args=None):
         if replace_address is None:
             replace_address = self.replaced_node.address()
+        if data_center is None:
+            data_center = "dc1"
```
Comment (ekaterinadimitrova2) [inline replace_address_test.py:82]: Why "dc1" was not added deterministically everywhere? What do I miss?

Comment (ekaterinadimitrova2) [reply replace_address_test.py:82]: Sorry, I just saw this change after I pressed approve

Comment (bereng) [reply replace_address_test.py:82]: Bc there is only one multi-dc test. The others work ok as they fallback to the default 'dc1' hard coded in the default topology file. But on multi-dc we need to be explicit.

**Comment by bereng** [comment]
@ekaterinadimitrova2 or @adelapena we'll need dtest runs for all versions. Would you be so kind to do that for me on jenkins please?

**Comment by adelapena** [comment]
Running dtests:
* [2.1](https://ci-cassandra.apache.org/view/all/job/Cassandra-devbranch-dtest/370/)
* [2.2](https://ci-cassandra.apache.org/view/all/job/Cassandra-devbranch-dtest/371/)
* [3.0](https://ci-cassandra.apache.org/view/all/job/Cassandra-devbranch-dtest/372/)
* [3.11](https://ci-cassandra.apache.org/view/all/job/Cassandra-devbranch-dtest/373/)
* [trunk](https://ci-cassandra.apache.org/view/all/job/Cassandra-devbranch-dtest/374/)

**Comment by bereng** [comment]
LGTM to my untrained eye on knows offenders for previous versions.

**Pull request closed without merging**